main.cpp

```diff
@@ -199,7 +231,39 @@ struct CarWash //2)
     2)power devices trough USB
     3)display the battery charge level
  */
+struct CarJumpstarter
```

Comment:
most jump starters are not limited to cars only, but to all vehicle types. 
- [ ] rename to `JumpStarter`

Reply:
ok, renamed CarJumpstarter to JumpStarter.